src/app/payment-form/payment-form.component.css

```diff
@@ -0,0 +1,64 @@
+.form-container {
+    background-color: #ffffff;
+    border: 1px solid #cccccc;
+    border-radius: 8px;
+    padding: 20px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    max-width: 600px;
+    margin: 0 auto;
+    margin-top: 50px;
+}
+
+h1 {
```

Comment:
to do list :
1.button global css file
2. create three type of button
3.move class loosely css
4.text size  and input size
5.make clear button secondary